### ISSUE TITLE
JSON5 getting picked up by the default parser. 

### ISSUE THREAD

**Issue by nicko-winner** [lukasoppermann/style-dictionary-utils#7]
The docs for this project reference a local path './parser/w3c-token-json5-parser', when converted to reference what looks like the appropriate file in 'style-dictionary-utils' the parser is not working on`.json5` files.

**Docs:**

```js
import StyleDictionary from 'style-dictionary-utils'
import { w3cTokenJson5Parser } from './parser/w3c-token-json5-parser'

StyleDictionary.registerParser(w3cTokenJson5Parser)
```

**Filed attempt:**
```js
const { w3cTokenJson5Parser } = require('style-dictionary-utils/dist/parser/w3c-token-json5-parser.js');
const StyleDictionary = BaseStyleDictionary.extend(options);
```

**Example stack blitz:**
https://stackblitz.com/edit/style-dictionary-example-yrpivg?file=build.mjs
to see error: `node ./build.mjs` to see that error (JSON5 files is run by standard parser).

**Failed work around:**
* adding parser to the `parsers` config property.

**Comment by lukasoppermann** [comment]
Hey, this should be fixed in [v1.1.2](https://github.com/lukasoppermann/style-dictionary-utils/releases/tag/v1.1.2)

Please reopen if the issue persists.

**Comment by nicko-winner** [comment]
Thanks Lucas!
